internal/build/imgsrc/buildkit_builder.go

```diff
@@ -81,6 +83,7 @@ func (r *BuildkitBuilder) Run(ctx context.Context, _ *dockerClientFactory, strea
 
 func (r *BuildkitBuilder) buildWithBuildkit(ctx context.Context, streams *iostreams.IOStreams, opts ImageOptions, dockerfilePath string, buildState *build) (i *DeploymentImage, err error) {
 	ctx, span := tracing.GetTracer().Start(ctx, "buildkit_build", trace.WithAttributes(opts.ToSpanAttributes()...))
+	ctx = appconfig.WithName(ctx, opts.AppName)
```

Comment:
Is it used by connectClient() later? I'd like to explicitly pass the app name. flyctl is passing too many in the context IMHO.

Reply:
changed in 44c7c449c71e8b0f57f6d1ea3bb5255e5c73ec4c :+1: